docs/how-to/analytics-dashboard.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Analytics Dashboard
+sidebar_label: Analytics Dashboard
+slug: /analytics-dashboard
+---
+
+# Analytics Dashboard
+
+Amplication's Analytics Dashboard is a powerful tool that provides developers with valuable insights into their application's development process.
```

Comment:
This feature is mostly for the "buyers" / managers and not for the developers. So maybe we should state "that provides developers and managers with..."